tests/test_integration.py

```diff
@@ -16,6 +17,15 @@ def json(self):
 def client():
     # Use our test integration config instead of the 'real' version
     file_path = find_dotenv('.env.test')
+
+    # Record existing env variable values
+    existing_values = {
+        "TRELLO_TOKEN": os.environ.get("TRELLO_TOKEN", ""),
+        "TRELLO_KEY": os.environ.get("TRELLO_KEY", ""),
+        "TRELLO_BOARD_NAME": os.environ.get("TRELLO_BOARD_NAME", ""),
+        "TRELLO_BOARD_ID": os.environ.get("TRELLO_BOARD_ID", ""),
+    }
```

Comment:
Nice, so you could run your tests in a single docker command if you wanted? But you chose not to?